watertap/data/techno_economic/CANDO_P.yaml

```diff
@@ -0,0 +1,51 @@
+default:
+  electricity_intensity_N:
+    value: 4
+    units: kWh/kg
+  oxygen_nitrogen_ratio:
+    value: 3.5
+    units: dimensionless
+  recovery_frac_mass_H2O:
+    value: 1
+    units: dimensionless
+  default_removal_frac_mass_solute:
+    value: 0
+    units: dimensionless
+  removal_frac_mass_solute:
+    phosphates:
+      value: 0
+      units: dimensionless
+    nitrogen:
+      value: 0
+      units: dimensionless
+    bcp:
```

Comment:
What is bcp?

Reply:
I see... bioconcentrated phosphorus. I would rename `bcp` to that name, which is more in line with how we have named other components.

Reply:
I can rename BCP.

> I'm also not clear on the difference between "phosphates" and "bioconcentrated phosphorus" when considering the reaction that's taking place.

The spreadsheet I have lists phosphorous (liquid) and BCP (solid) as different components. Phosphorous is converted into BCP via reaction. I have actually used the component "phosphates" instead of "phosphorous" in the model because you and @andrewlee94 have cautioned me against adding additional components to the code base if it might not be necessary.

Reply:
Ah, thanks! I didn't realize it was meant to represent phosphorus in solid phase. That makes sense.